### PR TITLE
Tickets/dm 22689

### PR DESCRIPTION
**Pull request by patrickingraham**
You get to have your revenge on how to write an example notebook now!
Note that this will need rebasing after your branch gets merged, but we can just move your examples to the appropriate directory.

### CODE REVIEW

File: examples/README [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The directories located here are used to differentiate between the types of notebooks that one may use for working with LSST. Short descriptions of each directory are as follows:
```
Comment (tribeiro) [inline examples/README]: So, if you rename this file `README.rst` GitHub will render it using rich text and they will look nicer. see https://github.com/lsst-ts/ts_notebooks/blob/tickets/DM-22689/README.rst

Also, can you rephrase this a little bit? Took me some time to figure out what it means. Maybe something like:

> This directory contains example notebooks on how to interact with the observatory control system. Notebooks are categorized and separated based on the their functionality. 
>
>Here is a short descriptions of each sub-directory:

Comment (patrickingraham) [reply examples/README]: Completed. The README is now more comprehensive... and most importantly it looks prettier :-)
